spec/spec_helper.rb

```diff
@@ -16,3 +16,20 @@
     end
   end
 end
+
+# set some constants to keep it DRY
+DEFAULT_VERSION = '8.13.5'
+PRODUCT_VERSION_STRING="atlassian-jira-software-#{DEFAULT_VERSION}"
+PRODUCT_VERSION_STRING_CORE = "atlassian-jira-core-#{DEFAULT_VERSION}"
+
+PATH_JAVA_HOME='/opt/java'
+PATH_JIRA_DIR='/opt/jira'
+PATH_INSTALLATION_BASE = "#{PATH_JIRA_DIR}/#{PRODUCT_VERSION_STRING}-standalone"
+
+FILENAME_SETENV_SH = "#{PATH_INSTALLATION_BASE}/bin/setenv.sh"
+FILENAME_USER_SH = "#{PATH_INSTALLATION_BASE}/bin/user.sh"
+FILENAME_CHECK_JAVA_SH = "#{PATH_INSTALLATION_BASE}/bin/check-java.sh"
+FILENAME_SERVER_XML = "#{PATH_INSTALLATION_BASE}/conf/server.xml"
+FILENAME_DBCONFIG_XML = '/home/jira/dbconfig.xml'
+FILENAME_CLUSTER_PROPS='/home/jira/cluster.properties'
+FILENAME_JIRA_CONFIG_PROPS='/home/jira/jira-config.properties'
```

Comment:
`spec_helper.rb` is managed automatically by modulesync. 
if you want to modify it you need to change `.sync.yml`. Take a look at [this](https://github.com/voxpupuli/puppet-zabbix/blob/master/.sync.yml#L8) example.

Also maybe it's better to put these into a file and require it from the spec_helper.
If you need these attributes to be available in the acceptance tests as well you can place it in a file in the directory `spec/support/acceptance`, this will automatically loaded by the spec_acceptance_helper